lxd/storage_volumes_utils.go

```diff
@@ -240,7 +245,7 @@ func storagePoolVolumeUsedByGet(s *state.State, volumeName string, volumeTypeNam
 	}
 
 	if len(volumeUsedBy) == 0 && len(profiles) == 0 {
-		return []string{}, err
+		return []string{}, os.ErrNotExist
```

Comment:
Same as above

Reply:
I don't like this checking for empty entities kind of stuff but as I said I'm fine with both.